Update msal-net-web-browsers.md (#88259)
Azure/azure-content: articles/active-directory/develop/msal-net-web-browsers.md

> ### Linux and MAC

Reviewer: ```suggestion
### Linux and macOS
```

> On Linux, MSAL.NET will open the default OS browser using the a tool such as `xdg-open`. Opening the browser as `sudo` does is not supported, and MSAL will throw an exception. On Mac, the browser is opened by invoking `open <url>`.

Reviewer: ```suggestion
On Linux, MSAL.NET opens the default OS browser with a tool like [xdg-open](http://manpages.ubuntu.com/manpages/focal/man1/xdg-open.1.html). Opening the browser with `sudo` is unsupported by MSAL and will cause MSAL to throw an exception.

On macOS, the browser is opened by invoking `open <url>`.
```

Author: Not sure about "root privileged like `sudo`". There is only `sudo` on Linux.

Reviewer: OK, still need some clarity on your original statement that mentions `sudo`.

When executed without `-u`, the _usual_ default behavior of `sudo` is to execute the command as root, and thus, with root privileges.

From the `sudo` man page:

```console
...
DESCRIPTION
     sudo allows a permitted user to execute a command as the superuser or another user, as specified by the security policy.  The invoking user's real (not
     effective) user-ID is used to determine the user name with which to query the security policy.
...
      -u user, --user=user
                 Run the command as a user other than the default target user (usually root).
```

This in particular is the confusing part of your original change:

> Opening the browser as `sudo` does is not supported

With the "does" in there, it sounds like you're saying that's _all_ that `sudo` does, ever, which is incorrect. I was trying to clarify what behavior of `sudo` in this situation was the most important, so if you have another way to do so, I'm all ears. :slightly_smiling_face:

Reviewer: @bgavrilMS How do you want to handle this one? Since opening the browser isn't _only_ thing `sudo` does, how about this:

> On Linux, MSAL.NET opens the default OS browser with a tool like [xdg-open](http://manpages.ubuntu.com/manpages/focal/man1/xdg-open.1.html). Opening the browser with `sudo` is unsupported by MSAL and will cause MSAL to throw an exception.

I've updated the suggestion above 👆 to reflect the newly suggested text for that line.

Author: That sounds good, thanks @mmacy for the update, this fell off my radar.

### Customizing the experience

> MSAL.NET is able to respond with an HTTP message or with an HTTP redirect when a token is received or in case of error. 

Reviewer: ```suggestion
MSAL.NET can respond with an HTTP message or HTTP redirect when a token is received or an error occurs.
```
